ducktape/command_line/main.py

```diff
@@ -24,10 +23,15 @@
 import argparse
 import os
 import sys
-
+import importlib
+import itertools
 
 def parse_args():
-    """Parse in command-line options.
+    """Parse in command-line and config file options.
+
+    Command line arguments have the highest priority, then user configs specified in ~/.ducktape/config, and finally
```

Comment:
This is the basic idea. Think about someone working on Kafka + Samza. It sucks to have to duplicate configs (like what cluster type to use) across multiple projects. (Although this also makes one of the drawbacks of the way we currently configure the Vagrant clusters pretty clear too -- it's not possible to share them between projects.)